net-mgmt/xymon-client/src/opnsense/mvc/app/models/Kumy/XymonClient/Settings.xml

```diff
@@ -0,0 +1,15 @@
+<model>
+    <mount>//kumy/xymon/client</mount>
+    <description>Xymon client settings</description>
+    <version>0.0.0</version>
+    <items>
+        <enabled type="BooleanField">
+            <default>1</default>
+            <Required>Y</Required>
+        </enabled>
+        <XYMSERVERS type="TextField">
```

Comment:
you probably want to validate the input in some way (depending on which options are valid)

Reply:
I'll switch to `HostnameField`